src/components/AccountWarning/AccountWarning.tsx

```diff
@@ -54,6 +54,25 @@ export function AccountWarning() {
         }
     }, [user, user?.has_active_warning_flag]);
 
+    const mainGoban = (window as any)["global_goban"];
+
+    if (location.pathname.indexOf("game/") > 0) {
+        // On the Game page, we need to delay displaying the warning until the game is over
+        if (liveGameInProgress) {
+            return null;
+        } else if (mainGoban?.engine.phase === "play" && mainGoban.engine.game_id) {
+            setLiveGameInProgress(true);
+            const checking = setInterval(() => {
```

Comment:
I think this works, but generally more idiomatic to call `setInterval()` inside a `useEffect`.

See Dan Abramov's comment on this [SO post](https://stackoverflow.com/a/64274431/5001502):

> Note that the correct way to do this is still `useEffect`. I'm clarifying for the next reader who might assume that this is the correct way to do intervals (it's not).

Reply:
👍 I'll have to read that post about "why  is this so".   But far be it from me to buck the idiom!